packages/core/src/pages/_document.tsx

```diff
@@ -16,6 +16,7 @@ function Document() {
           rel="dns-prefetch"
           href={`https://${storeConfig.api.storeId}.vtexassets.com`}
         />
+        <meta name="storefront" content="fast_store" />
```

Comment:
I don't think this underline makes sense here since we always use `faststore` in all places.

Reply:
Yeah, I have the same opinion. But this value needs to follow the pattern of the analytics team, and then use snake_case, fast_store is an existent index.